lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -149,6 +152,17 @@ class AwsCompileKafkaEvents {
         hasKafkaEvent = true;
         const { topic, batchSize, maximumBatchingWindow, enabled, consumerGroupId } = event.kafka;
         const startingPosition = event.kafka.startingPosition || 'TRIM_HORIZON';
+        const startingPositionTimestamp = event.kafka.startingPositionTimestamp;
+
+        if (
+          startingPosition === 'AT_TIMESTAMP' &&
+          !(startingPositionTimestamp !== undefined && startingPositionTimestamp !== null)
```

Comment:
Let's keep it simple as `startingPosition === 'AT_TIMESTAMP' && startingPositionTimestamp == null`

Reply:
nice